src/server/init.rs

```diff
@@ -246,14 +248,24 @@ pub fn init(options: &Init) -> anyhow::Result<()> {
         match method.bootstrap(&settings) {
             Ok(()) => {}
             Err(e) => {
-                log::error!("Bootstrap error, cleaning up...");
-                method.clean_storage(&settings.storage)
-                    .map_err(|e| {
-                        log::error!("Cannot clean up storage {}: {}",
-                            settings.storage.display(), e);
-                    }).ok();
-                Err(e).context(format!("cannot bootstrap {}",
-                                       settings.storage.display()))?
+                if e.is::<CannotCreateService>() {
+                    eprintln!("Error: {:#}", e);
+                    eprintln!("Bootrapping complete, \
+                        but there was an error creating a service. \
```

Comment:
Ditto, I think it should be `the server` in this context. @elprans ?

Reply:
I think we tried to create **a service** but failed to. So this there is no **the service** as it wasn't created. No?

Reply:
Well, depends on what kind of service the tool failed to create. If it was a systemd service config I'd say we should use "the". If it was a temporary service just to check if everything was installed it should be "a". In this context it sounds like "the".

**That said**, using articles correctly, unfortunately, isn't my strong skill :/  So I'd fix both articles to "the" and let @Dhghomon do a post-commit review :)

Reply:
Done

Reply:
Feels like "the" to me as well: it can still be multiple services but in the moment we're looking at "the" service in question. Also, using "a" feels (just a bit) like creating some new service that we didn't intend to.